widget/check.go

```diff
@@ -74,6 +74,17 @@ type Check struct {
 	OnChanged func(bool) `json:"-"`
 }
 
+// SetChecked sets the the checked state and refreshes widget
+func (c *Check) SetChecked(checked bool) {
```

Comment:
We may want to change the behavior for when the state is not actually changed (c.Checked == checked). I would consider adding something along the lines of:
```
if checked == c.Checked {
	return
}
```

This will eliminate unnecessary refreshes when programmatically resetting things (e.g. resetting a form). It will also avoid sending `OnChanged` events when nothing is actually changed.

Reply:
You are right, I updated the pull request.